### ISSUE TITLE
[ Feature ] Speed control video

### ISSUE THREAD

**Issue by castorinop** [meandavejustice/min-vid#391]
Like in youtube, can handle speeds 0.5, 0.25, 1, 1.25, 1.5, 2 with a keyboard shrortcut (shift + < or >). 

regards.

**Comment by gusmonod** [comment]
Yes, controlling the speed settings on the mini-player would be great!

Even if the control has to be done on the full-fledged player, it would be nice to have the mini-player keep the speed settings.
